packages/theme/composables/useCurrency/index.ts

```diff
@@ -32,8 +32,6 @@ const useCurrency = (): UseCurrency => {
 
   const change = (params: ComposableFunctionArgs<{ id: string }>) => {
```

Comment:
Since it doesn't support custom queries, we can use just the literal type here.

```suggestion
  const change = (params: { id: string }) => {
```